core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/CompactEntityComponentSpec.scala

```diff
@@ -1313,6 +1313,35 @@ class CompactEntityComponentSpec extends TestDriverComponentWithFlatSpecAndMatch
     }
   }
 
+  behavior of "listEntities"
+
+  it should "handle entities with no attributes" in withMinimalTestDatabase { _ =>
```

Comment:
I don't think this tests anything different than `handle entities with simple attributes` below. I would delete it.